File: Benchmarks_MCS-IIS/Programs_Benchmarks_MCS-IIS_BugAssist/0_Conditions_Fausses/ExempleMCSCardinalite2.c


 
   
    
void ExempleMCSCardinalite2(int i){int v,w,z;i=0;
  v=5;
  w=6;
  z=i+(v+w); // error in the assignment : z=i+(v+w); instead of z=i*(v+w);
  assert((z==0 || i!=0) && (v>=0) && (w>=0));
}

File: Benchmarks_MCS-IIS/Programs_Benchmarks_MCS-IIS/0_Conditions_Fausses/ExempleMCSCardinalite2.java
class programme{

 /*@ ensures 
   @ ((z==0 || i!=0) && (v>=0) && (w>=0));
   @*/ 
void foo(int i){
  int v=5;
  int w=6;
  int z=i+(v+w); // error in the assignment : z=i+(v+w); instead of z=i*(v+w);
}
}
